Model/src/GamePackage/Player.java

```diff
@@ -1,269 +0,0 @@
-package GamePackage;
```

Comment:
Is this the file that we needed to delete?

Reply:
yes